torch/nn/functional.py

```diff
@@ -1153,6 +1153,41 @@ def batch_norm(input, running_mean, running_var, weight=None, bias=None,
     )
 
 
+# This wrapper is for ONNX exporting.
+def instance_norm_wrapper(input, weight, bias, saved_running_mean, saved_running_var,
+                          training=False, momentum=0.1, eps=1e-5, affine=False):
+    import torch
+    func = instance_norm
+    if torch._C._jit_is_tracing(input):
+        import torch.onnx.symbolic
+        func = torch.onnx.symbolic.instance_norm_symbolic_builder()(instance_norm)
+    return func(input, weight, bias, saved_running_mean=saved_running_mean, saved_running_var=saved_running_var,
+                training=training, momentum=momentum,
+                eps=eps, affine=affine)
+
+
+def instance_norm(input, weight, bias, saved_running_mean, saved_running_var,
```

Comment:
Nice! Doesn't the functional wrapper need a new docstring?

Reply:
Sure, it should be helpful.